utils/constants.ts

```diff
@@ -112,147 +112,147 @@ export const chains: ChainProps[] = [
     address: "",
     chain: ChainKey.ARBITRUM,
     decimals: 18,
-    isDefault: true,
+    id: "0x1",
```

Comment:
if this is chain-id , then I don't think arbitrum is 0x1
cc: https://chainlist.org/